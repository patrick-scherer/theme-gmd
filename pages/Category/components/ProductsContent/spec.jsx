import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import { createMockStore } from '@shopgate/pwa-common/store';
import reducers from 'Pages/reducers';
import ProductsContent from './index';

const store = createMockStore(reducers);
jest.mock('../../../../components/View/context.js');

describe('<ProductsContent />', () => {
  it('should render', () => {
    const wrapper = mount((
      <Provider store={store}>
        <ProductsContent categoryId="1234" />
      </Provider>
    ));
    expect(wrapper).toMatchSnapshot();
  });
});
