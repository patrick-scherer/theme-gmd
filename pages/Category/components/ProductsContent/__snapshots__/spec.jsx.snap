// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductsContent /> should render 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <ProductsContent
    categoryId="1234"
    hasProducts={false}
  >
    <Portal
      name="product.list.before"
      props={
        Object {
          "categoryId": "1234",
        }
      }
    />
    <Portal
      name="product.list"
      props={
        Object {
          "categoryId": "1234",
        }
      }
    />
    <Portal
      name="product.list.after"
      props={
        Object {
          "categoryId": "1234",
        }
      }
    />
  </ProductsContent>
</Provider>
`;
